integration/simulation/simulation_in_mem_test.go

```diff
@@ -36,7 +36,7 @@ func TestInMemoryMonteCarloSimulation(t *testing.T) {
 		IsInMem:                    true,
 		L1TenData:                  &params.L1TenData{},
 		ReceiptTimeout:             5 * time.Second,
-		StoppingDelay:              4 * time.Second,
+		StoppingDelay:              15 * time.Second,
```

Comment:
turns out there was not enough time for the non-p2p node to catch up under some circumstances.